go.mod

```diff
@@ -12,7 +12,6 @@ require (
 	github.com/manifestival/controller-runtime-client v0.4.0
 	github.com/manifestival/manifestival v0.7.2
 	github.com/minio/minio-go/v7 v7.0.56
-	github.com/onsi/ginkgo/v2 v2.9.5
 	github.com/onsi/gomega v1.27.7
```

Comment:
gomega could be taken out from here as well, just make sure to run a `go mod tidy` locally.

Reply:
Done.